ui/src/app/views/homepage/component.tsx

```diff
@@ -281,13 +282,23 @@ export class HomepageComponent implements OnInit, OnDestroy {
         this.firstVisit = !profile.pageVisits.some(v =>
         v.page === HomepageComponent.pageId);
       }
-      this.serverConfigService.getConfig().subscribe((config) => {
-        if (environment.enableComplianceLockout && config.enforceRegistered) {
-          this.eraCommonsLinked = !!profile.linkedNihUsername;
-        } else {
-          this.eraCommonsLinked = true;
-        }
-      });
+
+      // Set Access Tasks flags
+      // TODO RW-1184 set trainingCompleted flag
+      this.eraCommonsLinked = !!profile.linkedNihUsername;
+
+      if (this.route.snapshot.queryParams.workbenchAccessTasks) {
+        // To reach the access tasks component from dev use /?workbenchAccessTasks=true
+        this.accessTasksRemaining = true;
+      } else {
+        this.serverConfigService.getConfig().subscribe((config) => {
+          if (environment.enableComplianceLockout || config.enforceRegistered) {
```

Comment:
&&

Reply:
good catch. changed that during the demo